tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AspNetCore/UserEvents/SignInManagerPasswordSignInUserIntegration.cs

```diff
@@ -101,7 +107,10 @@ internal static TReturn OnAsyncMethodEnd<TTarget, TReturn>(TTarget instance, TRe
                 if (security.IsAnonUserTrackingMode)
                 {
                     var anonId = UserEventsCommon.GetAnonId(id);
-                    tryAddTag(Tags.User.Id, anonId);
+                    if (anonId != null)
```

Comment:
same as above

Reply:
yes, fixed in [8fa918f](https://github.com/DataDog/dd-trace-dotnet/pull/6225/commits/8fa918f63e234e07613f17ed1634f12fdb342a0e)